src/index.html

```diff
@@ -9,10 +9,54 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The class `calendar--start-day-sun` suggests that the calendar starts on Sunday. Verify that the SCSS styles are consistent with this starting day to ensure the calendar displays correctly.